sway-core/tests/sway_to_ir/let_reassign_while_loop.ir

```diff
@@ -15,18 +15,18 @@ script {
 
         while_body:
         v4 = get_ptr mut ptr bool a, ptr bool, 0, !4
-        v5 = load ptr v4, !4
-        cbr v5, block0, block1, !5
+        v5 = get_ptr mut ptr bool a, ptr bool, 0, !5
```

Comment:
I think these changes are due to the fact that we're not creating `get_ptr` before compiling the RHS.

Reply:
OK, `v4` is used way down below for the `store` in `block1`, where the old `get_ptr` was, which is a bit unfortunate.